submissions/israelOladosu/database/transportation.sql

```diff
@@ -0,0 +1,84 @@
+CREATE DATABASE transporation;
+
+use transporation;
+
+-- classes table
+CREATE TABLE classes (
+    id INT NOT NULL AUTO_INCREMENT,
+    class_level INT NOT NULL,
+    PRIMARY KEY(id)
+);
+
+-- passangers table
+CREATE TABLE passengers (
+    id INT NOT NULL AUTO_INCREMENT,
+    firstname VARCHAR(50) NOT NULL,
+    lastname VARCHAR(50) NOT NULL,
+    sex VARCHAR(6) NOT NULL,
+    ticket_number VARCHAR(20) NOT NULL,
+    age INT,
+    class_id INT NOT NULL,
+    PRIMARY KEY(id),
+    FOREIGN KEY(class_id) REFERENCES classes(id) ON DELETE CASCADE 
+);
+
+CREATE TABLE cabin_classes(
+    id INT NOT NULL AUTO_INCREMENT,
+    cabin_code VARCHAR(10) NOT NULL,
+    passenger_id INT NOT NULL,
```

Comment:
Instead of passenger_id, this should reference `trips` table. As a trip which belongs to a passenger can have many cabins

Reply:
Hmm... Thank you, I get.